celery/backends/redis.py

```diff
@@ -181,6 +181,9 @@ def __init__(self, host=None, port=None, db=None, password=None,
 
         socket_timeout = _get('redis_socket_timeout')
         socket_connect_timeout = _get('redis_socket_connect_timeout')
+        retry_on_timeout = _get('redis_retry_on_timeout')
+        socket_keepalive = _get('redis_socket_keepalive')
+        health_check_interval = _get('redis_backend_health_check_interval')
```

Comment:
I see what we are doing here... I guess updating to a newer version of celery was not an option, right @javiertejero?

Reply:
right! well, 1) the latest version that contains the setting that we want is still pre-release (and I don't consider that a stable version...) 2) It is apparently a low risk backport compared to the risk and effort that involves the upgrade even if it get's released  ... 

Despite of this, yes, we aim to upgrade soon-ish